src/box/lua/misc.cc

```diff
@@ -213,6 +213,17 @@ port_msgpack_dump_lua(struct port *base, struct lua_State *L, bool is_flat)
 		luamp_decode(L, luaL_msgpack_default, &args);
 }
 
+/** Generate unique id for non-system space. */
+static int
+lbox_generate_space_id(lua_State *L)
+{
+	uint32_t ret = 0;
+	if (box_generate_space_id(&ret) != 0)
+		return luaT_error(L);
+	lua_pushinteger(L, ret);
```

Comment:
Nit: `lua_pushinteger()` works with 32-bit signed integers. Even though a space id shouldn't exceed `BOX_SPACE_MAX`, which equals `INT32_MAX`, it's still defined as `uimnt32_t` so it's better to use `lua_pushnumber()` instead.

Reply:
Changed to `lua_pushnumber`